### PR TITLE
Add documentation, example, and make code for defaultTraceHandler more readable

### PR DESCRIPTION
**Pull request by Geod24**
[This doc is pretty poor](https://dlang.org/phobos/core_runtime.html#.defaultTraceHandler)

### CODE REVIEW

File: src/core/runtime.d [hunk 1]
```diff
@@ -722,301 +773,242 @@ Throwable.TraceInfo defaultTraceHandler( void* ptr = null )
...
+        version (Posix)
+        {
+            // NOTE: The first 4 frames with the current implementation are
+            //       inside core.runtime and the object code, so eliminate
+            //       these for readability.  The alternative would be to
+            //       exclude the first N frames that are in a list of
+            //       mangled function names.
+            enum FIRSTFRAME = 4;
+        }
+        else version (Windows)
```
Comment (jacob-carlborg) [inline src/core/runtime.d]: Isn't Posix only now?

**Comment by Geod24** [comment]
Of course it breaks Mecca's hacks >.<

**Comment by Geod24** [comment]
Submitted https://github.com/weka-io/mecca/pull/21

File: src/core/runtime.d (hunk 1 above)
Comment (Geod24) [reply src/core/runtime.d]: That looks like a leftover. Removed.

**Comment by Geod24** [comment]
Mecca PR merged and a new tag has been pushed, ready to go

**Comment by Basile-z** [comment]
> Mecca PR merged and a new tag has been pushed, ready to go

Yes but testing on stable branches is still affected. Can you pick it ?

**Comment by Geod24** [comment]
I actually used the wrong version number in that PR 🤦‍♂ 
https://github.com/weka-io/mecca/pull/23

**Comment by Basile-z** [comment]
There's no emergency since we are in a beta phase but maybe it would be nice if this could be fixed before the next point release since now we put bug fixes more frequently in stable.

**Pull request merged**